src/Engine/CombatParameters.java

```diff
@@ -43,9 +43,9 @@ public void swap() {
 	
 	private void calculateParameters() {
 		baseDamage			= attacker.getDamage(defender);
-		attackFactor		= attacker.model.COStr;
+		attackFactor		= attacker.model.getDamageRatio();
```

Comment:
Had I remembered that we had a COStr variable already, I probably would have named the function [get/modify]COStrength(), but really either name will work.